README.md

```
@@ -1 +1,3 @@
-# github-chore
\ No newline at end of file
+# github-chore
+
+this repo contains chore github actions for rainlanguage org
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)